loadfile/loadfile_test.go

```diff
@@ -21,21 +21,20 @@ func Test_LoadContext(t *testing.T) {
 	assert.NoError(t, os.MkdirAll(testdir, os.ModePerm))
 
 	// create a directory
-	dirname := "mydir"
-	dirpath := filepath.Join(testdir, dirname)
-	assert.NoError(t, os.MkdirAll(dirpath, os.ModePerm))
+	dirPrefix := "mydir"
+	dirPath, err := os.MkdirTemp(testdir, dirPrefix+"*")
+	assert.NoError(t, err)
 
 	// create a file
-	filename := "myfile"
-	filePath := filepath.Join(testdir, filename)
-	f, err := os.Create(filepath.Clean(filePath))
+	filenamePrefix := "myfile"
+	f, err := os.CreateTemp(dirPath, filenamePrefix+"*")
 	assert.NoError(t, err)
+	filePath := f.Name()
 	assert.NoError(t, f.Close())
 
 	// create symlink to the directory
-	symlinkDirname := dirname + "_sym"
-	symlinkDirpath := filepath.Join(testdir, symlinkDirname)
-	assert.NoError(t, os.Symlink(dirpath, symlinkDirpath))
+	symlinkDirname := dirPath + "_sym"
+	assert.NoError(t, os.Symlink(dirPath, symlinkDirname))
```

Comment:
Is `testdir` the current default directory? Otherwise, it seems you've moved the symlink. Is that OK?